pkg/cover/canonicalizer.go

```diff
@@ -71,9 +75,11 @@ func (can *Canonicalizer) NewInstance(modules []host.KernelModule) *Canonicalize
 	instToCanonicalMap := make(map[uint32]*canonicalizerModule)
 	canonicalToInstMap := make(map[uint32]*canonicalizerModule)
 	for _, module := range modules {
+		discard := false
 		canonicalModule, found := can.modules[module.Name]
 		if !found || canonicalModule.Size != module.Size {
-			log.Fatalf("kernel build has changed; instance module %v differs from canonical", module.Name)
+			log.Logf(0, "kernel build has changed; instance module %v differs from canonical", module.Name)
```

Comment:
`log.Errorf` might be a better choice here (it won't kill the instance).

Reply:
Done